### PR TITLE
Annual grammar check 👀

### PR DESCRIPTION
**Pull request by Bluscream**
Fixed some typos and commented some stuff

### CODE REVIEW

File: TS3Client/Full/Packet.cs [hunk 1]
```diff
@@ -103,7 +103,7 @@ public Packet(ReadOnlySpan<byte> data, PacketType packetType, ushort packetId, u
 
 		public override string ToString()
 		{
-			return $"Type: {PacketType}\tFlags: [ " +
+			return $"Type: {PacketType}\tFlags: [ " + // TODO: Maybe use stringbuilder?
```
Comment (Splamy) [inline TS3Client/Full/Packet.cs]: No, thats a debug method, and supposed to be easily editable, not performant.

File: TS3Client/Full/Ts3Crypt.cs [hunk 2]
```diff
@@ -327,7 +327,7 @@ internal E<string> CryptoInit2(string license, string omega, string proof, strin
 
 			// Verify that our connection isn't tampered with
 			if (!VerifySign(serverPublicKey.Value, licenseBytes.Value, proofBytes.Value))
-				return "The init proof is not valid. Your connection might be tampered with or the server is an idiot.";
+				return "The init proof is not valid. Your connection might be tampered with or the server is an idiot."; // TODO: Do we really need this insult?
```
Comment (Splamy) [inline TS3Client/Full/Ts3Crypt.cs]: yes, the server is an idiot in that case.

File: TS3Client/Helper/Util.cs [hunk 3]
```diff
@@ -56,7 +56,7 @@ public static bool IsLinux
 
 	internal sealed class MissingEnumCaseException : Exception
 	{
-		public MissingEnumCaseException(string enumTypeName, string valueName) : base($"The the switch does not handle the value \"{valueName}\" from \"{enumTypeName}\".") { }
+		public MissingEnumCaseException(string enumTypeName, string valueName) : base($"The the switch does not handle the value \"{valueName}\" from \"{enumTypeName}\".") { } // Todo: "The the"?
```
Comment (Splamy) [inline TS3Client/Helper/Util.cs]: yup second 'the' is superfluous

File: TS3Client/TsDnsResolver.cs [hunk 4]
```diff
@@ -261,7 +261,7 @@ private static string ResolveNickname(string nickname)
 				return null;
 			}
 
-			return splits[0];
+			return splits[0]; // TODO: Maybe return all and try the next if one fails?
```
Comment (Splamy) [inline TS3Client/TsDnsResolver.cs]: That would maybe be nice but the current design structure does not allow for this.
Also iirc the dns protocol specifies that every ip should be valid, and the client should coose one.

File: TS3Client/Helper/Util.cs (hunk 3 above)
Comment (Bluscream) [reply TS3Client/Helper/Util.cs]: Edit's from maintainers are enabled ;)

File: TS3Client/TsDnsResolver.cs (hunk 4 above)
Comment (Bluscream) [reply TS3Client/TsDnsResolver.cs]: I get that... Maybe just keep it as todo for some far future

**Pull request merged**